ci-operator/config/openshift/telemeter/openshift-telemeter-master.yaml

```diff
@@ -44,7 +44,7 @@ tests:
   container:
     from: src
 - as: unit
```

Comment:
nit: we could rename it to `test` then?

Reply:
I didn't change the name because 1) rule tests are also unit tests and 2) (more important) I'm lazy :)

Reply:
Then let’s keep it for the next person who might get confused by it :)